README.md

```diff
@@ -219,7 +223,9 @@ hetzner-k3s create --config cluster_config.yaml
 
 This will take a few minutes depending on the number of masters and worker nodes.
 
-
+### disable public IPs (IPv4 or IPv6 or both) on nodes
+With `enable_public_net_ipv4: false` and `enable_public_net_ipv6: false` you can disable public interface at hetzner node creation. This settings are global and effects all master and worker. If you disable public IPs be sure that your hetzer-k3s executor has excess to the hetzner private network the nodes will be deployed in (executor is already in the privNet or has VPN access).
+Additional setup network of the nodes via cloud init, so you have internet excess and DNS. If not the cluster install process will stuck after creation all nodes via hetzner cloud.
```

Comment:
Could you provide an example how the cloud init is configured? Are you using a NAT gateway or a FW proxy?

Reply:
I opened a discussion for that: https://github.com/vitobotta/hetzner-k3s/discussions/252

Reply:
Very interesting. We are looking for a solution like this. 
BTW in order to make the k8s API Loadbalancer accessible with an alternative hostname, so you can point a DNS record to the private IP of the Loadbalancer, I have made this PR: https://github.com/vitobotta/hetzner-k3s/pull/251.

Reply:
@mgalesloot Nice one will check it out in the next time. We running our internal setup since some weeks in production it works fine and also save costs by disabling ext. IPv4 Addresses.